lib/helium/collection.rb

```diff
@@ -80,6 +80,27 @@ def last
       collection.last
     end
 
+    # Adds resources of the same type to the collection related to the
+    # '@belongs_to' resource
+    def add_relationships(items)
+      body = relationship_request_body(items)
```

Comment:
we might want to do something like `items = Array(items)` here, just in case someone passes in a single sensor